test-configs/operator-wandb/fmb.yaml

```diff
@@ -66,6 +72,34 @@ flat-run-fields-updater:
     requests:
       cpu: "100m"
       memory: "128Mi"
+  image:
+    tag: 0.70.0-metric-observer-available.0
+
+metric-observer:
+  install: true
+  pubSub:
+    subscription: "metric-observer"
+  env: 
+    # TODO: I assume SHADOW_QUEUE needs to be updated on a per-env basis?
```

Comment:
this was a question for @danielpanzella @dacbd

Reply:
GORILLA_RUN_UPDATE_SHADOW_QUEUE should itself be defined the same for al environments, so this should be in the values.yaml.  within it are two helm template references `{{ (include "wandb.bucket" . | fromYaml).url | quote}}` and `{{ include "wandb.runUpdateShadowQueue" . | quote }}` which will render the values appriopriate for the environment

Reply:
that makes sense, thanks! will make that change

Reply:
ah- I'll answer my own question. The template uses `pubsub.subscription`, and it gets that value from the metric-observer, so that's how we get the unique value. thanks!